group1/comm/templates/ChangePasswd.html

```diff
@@ -0,0 +1,125 @@
+                            {% endif %}
+                        </div>
+                    </div>
+                    {% if form.new_password2.errors %}
+                    <div class="form-group has-error has-feedback">
+                    {% else %}
+                    <div class="form-group">
+                    {% endif %}
+                        <label for="{{ form.new_password2.id_for_label }}" class="col-xs-12 col-sm-4 control-label">Confirm Password</label>
+                        <div class="col-xs-12 col-sm-8" id="{{ form.new_password2.id_for_label }}_popover">
+                            {{ form.new_password2 }}
+                            {% if form.new_password2.errors %}
+                            <span class="glyphicon glyphicon-remove form-control-feedback" aria-hidden="true"></span>
+                            <span id="inputError2Status" class="sr-only">(error)</span>
+                            {% endif %}
+                        </div>
+                    </div>
+	        	</div>
+        		<div class="modal-footer">
+        		  	<a class="btn btn-default" href="javascript:void(0);" onclick="closeCommDialog();">Close</a>
+        		  	<a class="btn btn-primary" href="javascript:void(0);" onclick="savePasswdChanges();">{{ button_desc }}</a>
+        		  	<!-- <button type="submit" class="btn btn-primary">{{ button_desc }}</button> -->
+        		</div>
+	        </form>
+
+      	</div><!-- /.modal-content -->
+    </div><!-- /.modal-dialog -->
+
+	<script type="text/javascript">
+        {% for field in form %}
+        {% if field.errors %}
+        var {{ field.id_for_label}}_errors = '';
+        {% for error in field.errors %}
+        {{ field.id_for_label }}_errors += '{{ error|escape }}';
+        {% endfor %}
+        $("#{{ field.id_for_label }}_popover").popover({
+            animation: true,
+            html: true,
+            content: {{ field.id_for_label }}_errors,
+            placement: 'right',
+            trigger: 'hover'
+        });
+        {% endif %}
+        {% endfor %}
+
+
+        // detect submit button event to use ajax method to post the form
+        // and retrive the return data back if there is error object in form object.
+        function savePasswdChanges() {
+            $.ajax({
+                type: "POST",
+                cache: false,
+                url: $("#changePwdForm").attr('action'),
+                data: $("#changePwdForm").serialize(),
+                success: function(data) {
+                    if (data != "") {
+                        $("#dialogModal").html(data);    
+                    } else {
+                        // location.reload(true);
+                        location.replace('/signin?next=' + location.pathname)
+                    };
+                    
+                },
+                async: true
+            });
+        }
+
+		// detect submit button event to use ajax method to post the form
```

Comment:
This whole block is commented.  Can it just be removed or do we need it for something?